test/unified_format.py

```diff
@@ -977,7 +977,6 @@ def _testOperation_targetedFailPoint(self, spec):
         client = single_client('%s:%s' % session._pinned_address)
         self.__set_fail_point(
             client=client, command_args=spec['failPoint'])
-        self.addCleanup(client.close)
```

Comment:
Instead of removing this, we should move it above `self.__set_fail_point`.

Reply:
Done.